src/kpatch_user.c

```diff
@@ -904,9 +908,11 @@ processes_do(int pid, callback_t callback, void *data)
 
 		rv = callback(pid, data);
 		if (rv < 0)
-			ret = -1;
-		if (rv == -2)
+			ret = rv;
+		if (rv == -2) {
```

Comment:
`rv=-1` is used to indicate that we found what we have been looking for. Please rename that to some define or constant and restore the old behavior.